generators/app/templates/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+<%= imageName %>:
+  dockerfile: Dockerfile.<%= environment %>
+  build: .<% if (isWebProject && portNumber) { %>
```

Comment:
If i put the newline outside of the tag it ends up in the generated file. Would you like one within the tag?

---
In reply to: [64833362](https://github.com/Microsoft/generator-docker/pull/57#discussion_r64833362) [](ancestors = 64833362)